torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -623,11 +623,25 @@ RegisterOperators reg(
          },
          aliasAnalysisSpecialCase()),
      // This operator is generated inside the compiler for indexing into
+     // ModuleList without a statically determinable key. Accordingly,
+     // self must be a ModuleType and the output must be an InterfaceType.
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA(
+             "prim::ModuleContainerIndex.list(Any self, int ind) -> Any"),
+         [](Stack* stack) {
+           IValue ind = pop(stack);
+           IValue module_dict = pop(stack);
+           std::stringstream ss;
+           ss << ind.toInt();
+           push(stack, module_dict.toModule().attr(ss.str()));
+         },
+         aliasAnalysisFromSchema()),
+     // This operator is generated inside the compiler for indexing into
      // ModuleDict without a statically determinable key. Accordingly,
      // self must be a ModuleType and the output must be an InterfaceType.
      OperatorGenerator(
          TORCH_SELECTIVE_SCHEMA(
-             "prim::ModuleDictIndex(Any self, str ind) -> Any"),
+             "prim::ModuleContainerIndex.dict(Any self, str ind) -> Any"),
```

Comment:
This is not BC, but I think I can make it BC by not changing the operator name, but making the `ind` parameter `Any` and using this operator for both `ModuleDict` and `ModuleList`. Thoughts?

Reply:
Based on my rough understanding, this doesn't affect serialized code. So it is not really BC-breaking I think. And AFAIK, we never made promise about operator backward-compatbility. Therefore, I think this is fine, let me know what you think.

Reply:
We only make operator-level BC promises about mobile & afaik this didnt work on mobile anyway, so this should be fine as @gmagogsfm said